tests/History.spec.js

```diff
@@ -0,0 +1,60 @@
+var History = require('../public/js/History.js');
+
+describe('History', function() {
+    var history;
+
+    beforeEach(function() {
+        history = new History();
+    });
+
+    describe('add', function() {
```

Comment:
Nice! Relates to my comment about the if statement... Yeah a note in the docstring or under the if would be very useful to avoid confusion in the future!